src/modules/displayShows.js

```diff
@@ -0,0 +1,6 @@
+/* eslint-disable import/prefer-default-export */
+export const fetchData = async (id) => {
+  const fetching = await fetch(`https://api.tvmaze.com/shows/${id}`)
+    .then((response) => response.json());
+  return fetching;
+};
```

Comment:
Kindly enable eslint for it is not allowed to disable linters. linters help us write clean code therefore it is against microverse policy to disable linters.

Reply:
I can't disable this because ESlint in this particular code forces us to use default export, and default export does not work with const declarations (arrow functions), If I used regular functions it will work fine but this is not the case because it is a requirement to use arrow functions as the previous code reviewer stated, also it's impossible to follow Eslint guidelines with Webpack, it will break the entire code since Webpack has different syntaxes mixed up.
##NOTE: In all of my previous projects I have disabled ESlint in these special cases and got approved no problems.